api/internal/managers/linux/infra/manager.go

```diff
@@ -175,3 +191,9 @@ func NewInfraManager(opts ...InfraManagerOption) *infraManager {
 func (m *infraManager) Get() (types.Infra, error) {
 	return m.infraStore.Get()
 }
+
+func (m *infraManager) UpdateConfigValues(configValues map[string]string) {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+	m.configValues = configValues
```

Comment:
this should be passed to the infraManager.Install function as an arg instead of updating a global variable like this